### ISSUE TITLE
Would you consider a "delete removed episode" option?

### ISSUE THREAD

**Issue by lardconcepts** [dvehrs/podget#30]
I use Podget for a rather niche edge-case of grabbing a podcasts and re-broadcasting them. It's a long story!

The 7 podcasts  might get added to twice a day, and when items pass their "valid until" date (if applicable), the pod-owner removes them. But they should NOT then be available from my server. So the normal "cleanup after x days" won't work.

Right now, I'm fudging it by deleting the entire tree of directories and doing a "--force" twice a day to refresh everything. Rather wasteful and slow.

I'm toying with the idea of using a python script called beautifulsoup to build an array of track titles from the xml, and then get another array of the files currently downloaded, and then just delete the difference between the two.

In other words, I have "_a solution_" which wouldn't take me too long to build, but I thought I'd ask here for to see if there's a better way, or even as a possible suggestion for a new podget feature!

So, to summarise: Could Podget be made to compare the files downloaded with the files in the current xml and delete the differing files OR re-download a file where the filename was the same but the size had changed?

Thanks.

**Comment by dvehrs** [comment]
Interesting idea.  However as you mention, this modification would be for a rather small set of potential users.  Rather than modify podget directly (and create all the potential support issues), I wonder if it would not be better to create a separate script that can be distributed with podget in the SCRIPTS directory that those who need can use.  

That just my first thoughts, I will let this bounce around the old noggin to see if any other potential implementations come to mind.   Or maybe someone else has a similar need and has some ideas?
